shared/src/androidMain/kotlin/com/kcc/kmmhackathon/shared/utility/DateFormattingUtil.kt

```diff
@@ -0,0 +1,14 @@
+package com.kcc.kmmhackathon.shared.utility
+
+import java.text.SimpleDateFormat
+
+actual class DateFormattingUtil actual constructor() {
+    actual fun convertTimeStamp(timeStamp: Long): String {
+        val date = java.util.Date(timeStamp)
+        return formatter.format(date)
```

Comment:
Let me know if there is a better way - I literally pulled this off of stackoverflow and it seems to work 🤷‍♂️

Reply:
Potentially you could use some ideas of @dsmith42 from [here](https://github.com/theappbusiness/benchHackathonNov20/pull/53/files#diff-b0440d59189aeee68196c532362b4f9bc04d7e4849c375acfb0310ad83c08c5c)

I believe if you could use the `dateStyle` and `timeStyle` as the same of the iOS side. You could define the format in a more readable way.

Reply:
Yeah, maybe a discussion for the whole team.
I just don't know how useful it will be to have an expiry date of "Mon 17 Nov 2020 17:05" displayed in the app?

Reply:
Maybe you could use RelativeDateTimeFormatter instead?

https://developer.android.com/reference/android/icu/text/RelativeDateTimeFormatter 
https://developer.apple.com/documentation/foundation/relativedatetimeformatter

Reply:
I'll check it out :-)

Reply:
My thinking about the time was that if I were to list some food, I may want people to pick it up by 9pm so would set that as the expiry.

Reply:
Ahh, I am seeing expiry as a Use By date, rather than a collection date.
We could probably make the case for both

Reply:
Yes, perhaps UseBy food date, and collect by (or even collection window).  I guess that is best for discussion as group